test/run

```diff
@@ -20,6 +20,9 @@ main() {
       '
 
   root=$PWD
+  if [ -z $kak_bin ]; then
```

Comment:
should quote the variable expansion here to avoid weird errors when it contains spaces.
Same below, or do I do want to allow `kak_bin='strace -o foo.strace kak'` like this?
Seems like what DEBUGGER is for.

Reply:
quoting seems appropriate for both usages of `kak_bin`, since I can't think of a use case where splitting the variable's value on spaces would be useful. It will always be a single path, and not a potential list of args.